ossm/values.yaml

```diff
@@ -18,7 +18,42 @@ csv:
   categories: OpenShift Optional, Integration & Delivery, Networking, Security
   capabilities: Seamless Upgrades
   description: The OpenShift Service Mesh Operator enables you to install, configure, and manage an instance of Red Hat OpenShift Service Mesh. OpenShift Service Mesh is based on the open source Istio project.
-  longDescription: TODO
+  longDescription: |-
+    Red Hat OpenShift Service Mesh is a platform that provides behavioral insight and operational control over a service mesh, providing a uniform way to connect, secure, and monitor microservice applications.
+
+    ### Overview
+
+    Red Hat OpenShift Service Mesh, based on the open source [Istio](https://istio.io/) project, adds a transparent layer on existing
+    distributed applications without requiring any changes to the service code. You add Red Hat OpenShift Service Mesh
+    support to services by deploying a special sidecar proxy throughout your environment that intercepts all network
+    communication between microservices. You configure and manage the service mesh using the control plane features.
+
+    Red Hat OpenShift Service Mesh provides an easy way to create a network of deployed services that provides discovery,
```

Comment:
should we have these line breaks or rather use free-floating text?

Reply:
It seems OLM is ignoring those line breaks:
![image](https://github.com/user-attachments/assets/44b0f3e8-317d-465d-b55c-4eefbdf8b562)

Reply:
@dgn ok to merge?